[2022-07-23 09:13:52,328] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: local_to_snowflake.upload_to_s3_2019-02 manual__2022-07-23T09:13:39.645249+00:00 [queued]>
[2022-07-23 09:13:52,347] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: local_to_snowflake.upload_to_s3_2019-02 manual__2022-07-23T09:13:39.645249+00:00 [queued]>
[2022-07-23 09:13:52,347] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-23 09:13:52,347] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-07-23 09:13:52,348] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-23 09:13:52,377] {taskinstance.py:1397} INFO - Executing <Task(LocalFilesystemToS3Operator): upload_to_s3_2019-02> on 2022-07-23 09:13:39.645249+00:00
[2022-07-23 09:13:52,395] {standard_task_runner.py:52} INFO - Started process 480 to run task
[2022-07-23 09:13:52,407] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'local_to_snowflake', 'upload_to_s3_2019-02', 'manual__2022-07-23T09:13:39.645249+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/localtosnowflake.py', '--cfg-path', '/tmp/tmp0sfj31le', '--error-file', '/tmp/tmp1tu17ecf']
[2022-07-23 09:13:52,411] {standard_task_runner.py:80} INFO - Job 53: Subtask upload_to_s3_2019-02
[2022-07-23 09:13:52,544] {task_command.py:371} INFO - Running <TaskInstance: local_to_snowflake.upload_to_s3_2019-02 manual__2022-07-23T09:13:39.645249+00:00 [running]> on host cbd64c50b1d6
[2022-07-23 09:13:52,718] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=local_to_snowflake
AIRFLOW_CTX_TASK_ID=upload_to_s3_2019-02
AIRFLOW_CTX_EXECUTION_DATE=2022-07-23T09:13:39.645249+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-23T09:13:39.645249+00:00
[2022-07-23 09:13:52,738] {base.py:68} INFO - Using connection ID 'aws-conn' for task execution.
[2022-07-23 09:13:52,739] {base_aws.py:206} INFO - Credentials retrieved from login
[2022-07-23 09:13:53,868] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=local_to_snowflake, task_id=upload_to_s3_2019-02, execution_date=20220723T091339, start_date=20220723T091352, end_date=20220723T091353
[2022-07-23 09:13:53,902] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-23 09:13:53,953] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
